rules/linux/lnx_system_info_discovery.yml

```diff
@@ -0,0 +1,48 @@
+action: global
+title: System Information Discovery
+id: 42df45e7-e6e9-43b5-8f26-bec5b39cc239
+status: stable
+description: Detects system information discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/08
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1082/T1082.md
+falsepositives:
+    - Legitimate administration activities
+level: low
+tags:
+    - attack.discovery
+    - attack.t1082
+---
+logsource:
+    product: linux
+    categories: process_creation
+detection:
+    selection:
+        Image|endswith:
```

Comment:
The ART test checks for specific values returned from these commands. Simple command execution is only a very weak signal.

Reply:
discussion here https://github.com/SigmaHQ/sigma/pull/1362#discussion_r598336458